logging/logfile_test.go

```diff
@@ -2,135 +2,120 @@ package logging
 
 import (
 	"io/ioutil"
+	"os"
 	"path/filepath"
 	"testing"
 	"time"
 
-	"github.com/hashicorp/consul/sdk/testutil"
-)
+	"github.com/stretchr/testify/require"
 
-const (
-	testFileName = "Consul.log"
-	testDuration = 50 * time.Millisecond
-	testBytes    = 10
```

Comment:
I removed this, because test constants like this can actually be harmful instead of helpful. 

1. it encourages tests which use the same values, instead of different values, which could lead to hidden bugs because only a single value is tested.
2. it makes tests less obvious because the input values are hidden away instead of explicit in the test case.